qkernel/src/syscalls/sys_proxy.rs

```diff
@@ -246,4 +647,77 @@ pub fn CudaMemcpy(task: &Task, dst: u64, src: u64, count: u64, kind: CudaMemcpyK
         _ => todo!()
     }
     
+}
+
+fn CudaMemcpyAsync(task: &Task, dst: u64, src: u64, count: u64, kind: CudaMemcpyKind, stream: u64) -> Result<i64> {
+    match kind {
+        CUDA_MEMCPY_HOST_TO_HOST => {
+            // error!("CudaMemcpy get unexpected kind CUDA_MEMCPY_HOST_TO_HOST");
```

Comment:
This error message shall be uncommented because it gives info to user for such error.